src/optee_test.c

```diff
@@ -33,6 +34,8 @@ TEEC_Context xtest_teec_ctx;
 
 char *xtest_tee_name = NULL;
 
+int level = 0;
```

Comment:
static?

Reply:
Thats a bit of ugly solution. But we have what we have. All test suites requires level to be set explicitly. Some of the tests requires level > 0. That's why static can't be used here - some tests may want to set different level.